scripts/package.json

```diff
@@ -0,0 +1,8 @@
+{
+  "private": true,
+  "devDependencies": {
+    "dedent": "^0.7.0",
+    "glob": "^7.1.2",
+    "markdown-magic": "^0.1.20"
+  }
```

Comment:
Let’s add a `scripts` section with a `build` command so people can run `npm run build` instead of `node scripts/generate.js`.

Reply:
It’s just `./scripts/generate.js` right now - the reason it’s this way is so the root of the repo doesn’t get cluttered with package.json and .npmrc. If that’s not an issue, i can move them all up, and then `npm run build` would work?

Reply:
Updated the OP with more explanation of this.